examples/main/main.cpp

```diff
@@ -760,7 +760,8 @@ int main(int argc, char ** argv) {
                         ? last_output.length() - static_cast<size_t>(antiprompt.length() + extra_padding)
                         : 0;
 
-                    if (last_output.find(antiprompt, search_start_pos) != std::string::npos) {
+                    auto tmp = ::llama_tokenize(ctx, antiprompt, false, true);
```

Comment:
Maybe pre-tokenize once at the start of the program, to avoid tokenization overhead in each iteration

Reply:
I think it should be better now.